org.adempiere.base/src/org/compiere/model/GridField.java

```diff
@@ -591,6 +599,13 @@ public void setInserting (boolean inserting)
 		m_inserting = inserting;
 	}   //  setInserting
 
+	public void setDefaultLogic(String defaultValue) {
+		m_vo.DefaultValue = defaultValue;
+	}
+
+	public void setDefault2Logic(String defaultValue2) {
+		m_vo.DefaultValue2 = defaultValue2;
+	}

```

Comment:
this actually not needed. getVO().DefaultValue and getVO().DefaultValue2 did the same thing.

Reply:
You're right, but I think this way is better and more intuitive. Trying to follow the law of Demeter here.

Reply:
agree but your ticket is misleading to implied that it is not possible with existing code.

Reply:
That's true. Just changed the commit message and the ticket description and title to reflect what you mean